src/main/java/de/tum/cit/aet/artemis/lecture/domain/LectureTranscription.java

```diff
@@ -0,0 +1,73 @@
+package de.tum.cit.aet.artemis.lecture.domain;
+
+import java.util.List;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Convert;
+import jakarta.persistence.Entity;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.Size;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import de.tum.cit.aet.artemis.core.domain.DomainObject;
+
+@Entity
+@Table(name = "lecture_transcription")
+public class LectureTranscription extends DomainObject {
```

Comment:
For consistency, it might make sense to specify column names using `@Column(name = <column-name>)`. In particular, for camel case it's just more explicit.